app/src/main/java/me/ccrama/redditslide/Reddit.java

```diff
@@ -248,18 +253,51 @@ public static String arrayToString(ArrayList<String> array, String separator) {
     public static Integer getSortingId(String subreddit) {
         Sorting sort =

```

Comment:
@freundTech awesome feature ! this is much cleaner than the the long list we had. Small bug, does not work with multireddits added to the sidebar. From what I can see, all we need is to make 'subreddit' all lowercase as all keys in 'sorting' map are lowercase. Same for the time also in the getSortingIdTime() method.